homeassistant/components/immich/strings.json

```diff
@@ -68,6 +68,11 @@
       "usage_by_videos": {
         "name": "Disk used by videos"
       }
+    },
+    "update": {
+      "update": {
+        "name": "Immich server"
```

Comment:
It's attached to a device, so I'm not sure we should add Immich to the name. What did we do with paperless?